apps/src/applab/commands.js

```diff
@@ -554,6 +554,19 @@ applabCommands.setStrokeWidth = function (opts) {
   return false;
 };
 
+// Returns an rbg or rgba color string that can be used as a parameter to other functions.
+applabCommands.rgb = function (opts) {
+  apiValidateTypeAndRange(opts, 'color', 'number', opts.r, 'number', 0, 255);
+  apiValidateTypeAndRange(opts, 'color', 'number', opts.g, 'number', 0, 255);
+  apiValidateTypeAndRange(opts, 'color', 'number', opts.b, 'number', 0, 255);
+  const colorString = `${opts.r}, ${opts.g}, ${opts.b}`;
+  if (opts.a) {
```

Comment:
does this work properly if you intend to call it with `a` = `0`? Seems like that might be falsey